src/api/app/views/webui/project/_basic_info.html.haml

```diff
@@ -0,0 +1,29 @@
+- if flipper_responsive?
+  .basic-info.mb-3{ class: 'in-place-edition' }
```

Comment:
The word is `editing`, edition is something else.

Reply:
Oh, good to know! I replace it in the code and as soon as we squash the commits I'll correct the commit's texts as well.